server/database/index.js

```diff
@@ -1,22 +1,117 @@
-const {Sequelize, DataTypes} = require('sequelize');
-const dialects = ['mysql', 'postgres'];
+const cassandra = require('cassandra-driver');
+const client = new cassandra.Client({ contactPoints: [`${process.env.DB_HOST}:${process.env.DB_PORT}`], localDataCenter: 'datacenter1', keyspace: 'options' });
 
-const db = new Sequelize(
-  'options', 'root', '', {
-    host: '127.0.0.1',
-    // can try port number
-    port: 3307,
-    dialect: 'mysql'
-  });
 
+module.exports.getOneProduct = (productId, callback) => {
```

Comment:
good job covering all the query cases. but since line 7-14 is a repeating step for nearly all operations, it could potentially be its own function?

Reply:
Great point! I will implement that.